### PR TITLE
Added version to the lib

### PR DESCRIPTION
**Pull request by alejandrojnm**
Added the option to get the version of the lib from the go.mod and add the function to set the user agent

Signed-off-by: Alejandro J. Nuñez Madrazo <alejandrojnm@gmail.com>

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -83,7 +86,7 @@ func NewClientWithURL(apiKey, civoAPIURL, region string) (*Client, error) {
 
 	client := &Client{
 		BaseURL:   parsedURL,
-		UserAgent: "civogo/" + Version,
+		UserAgent: "civogo/" + utils.GetVersion(),
```
Comment (RealHarshThakur) [inline client.go:89]: This is confusing. We're supposed to set `client.UserAgent := "civogo/"+utils.GetVersion()` and then `c.SetUserAgent(&component{})` separately?

File: client.go [hunk 2]
```diff
@@ -271,3 +274,12 @@ func (c *Client) DecodeSimpleResponse(resp []byte) (*SimpleResponse, error) {
 	err := json.NewDecoder(bytes.NewReader(resp)).Decode(&response)
 	return &response, err
 }
+
+// SetUserAgent sets the user agent for the client
+func (c *Client) SetUserAgent(component *Component) {
+	if component.ID == "" {
+		c.UserAgent = fmt.Sprintf("%s/%s %s", component.Name, component.Version, c.UserAgent)
```
Comment (RealHarshThakur) [inline client.go:281]: I think this should be : 
```suggestion
		c.UserAgent = fmt.Sprintf("%s/%s %s", component.Name, component.Version, utils.GetVersion())
```

File: client.go (hunk 1 above)
Comment (alejandrojnm) [reply client.go:89]: Will keep the civogo version in every request

File: client.go (hunk 2 above)
Comment (alejandrojnm) [reply client.go:281]: If we do it in this way we lost the `civogo` string

**Review by RealHarshThakur — APPROVED** [review]

**Pull request merged**